src/service-api.ts

```diff
@@ -105,9 +105,10 @@ const plugin: FastifyPluginAsync<GraaspPluginZipOptions> = async (fastify, optio
     }
 
     // create the items
-    const tasks = items.map((item) => iTM.createCreateTaskSequence(member, item, parentId));
+    const tasks = items.map((item) => {
+      return iTM.createCreateTaskSequence(member, item, parentId);
```

Comment:
style: is there a reason why this was changed to a bracketed return map style? IMO the previous value-expression is more concise and readable